helpers/github_helpers.rb

```diff
@@ -2,11 +2,13 @@ module GithubHelpers
 
   @@contributors = {}
   def build_contributors(repo)
-    return unless config[:github_auth]
+    throw "Authentication missing" if config[:github_auth].empty? or config[:github_tfa].empty?
```

Comment:
Also there is a runtime issue with error `undefined method `empty?' for false:FalseClass`